tests/full_matrix/full_matrix_determinant_01.cc

```diff
@@ -0,0 +1,93 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// Check that the determinant for a FullMatrix of
+// size greater than 3 is correct (i.e. as computed
+// by the Laplace expansion)
```

Comment:
I agree with @bangerth, I would not bother with anything that is not supported by Lapack.